src/instructlab/config/init.py

```diff
@@ -359,9 +375,12 @@ def get_params(
         cfg = read_config(config)
     clone_taxonomy_repo = True
     if interactive:
-        click.echo(
-            "Welcome to InstructLab CLI. This guide will help you to setup your environment."
-        )
+        click.echo()
+        click.echo("-" * 52)
```

Comment:
What happens when the terminal width is smaller than 52?

Reply:
the 52 * "-" is based on the sentence, so if smaller than 52, it will show it in next line. Actually, the long sentence is the same.
![image](https://github.com/user-attachments/assets/cbb61684-3a64-4101-a30a-872dd769efdf)

Reply:
The text wrapping is probably fine.  The -'s ideally wouldn't be wider than the window though.

Reply:
is it ok to get the width?
e.g.
```
    terminal_width = shutil.get_terminal_size().columns
    separator = "-" * min(52, terminal_width) 
```

Reply:
> is it ok to get the width?

I think so, just needs some testing across environments.

Reply:
It would also be good to calc the 52 based on the length of the string.